### PR TITLE
SAIS: Add recipe

### PR DESCRIPTION
**Pull request by win8linux**
[*Strange Adventures in Infinite Space*](https://rich.itch.io/strangeadventures) (SAIS) is one of the first roguelikes made with the genre of science fiction in mind. Now it comes to Haiku via this recipe, based on the [updated GPL source code](https://github.com/kuroneko/sais).

It should be noted that the game needs to be killed manually, as exiting it normally will just blank out the window.

The GPL v3 source code comes with the assets, which are under the different CC-BY-NA 4.0 license. Does that need to be indicated in the recipe in some way or is that unnecessary?

### CODE REVIEW

File: games-roguelike/sais/sais-1.6.3~git.recipe [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:zip
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir sais-build &&
```
Comment (Begasus) [inline games-roguelike/sais/sais-1.6.3~git.recipe:65]: Trailing whitespace here :)

File: games-roguelike/sais/sais-1.6.3~git.recipe [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	devel:libSDL2_mixer_2.0$secondaryArchSuffix
+	devel:libphysfs$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:zip
+	cmd:pkg_config$secondaryArchSuffix
+	"
```
Comment (Begasus) [inline games-roguelike/sais/sais-1.6.3~git.recipe:61]: Sort A-Z

File: games-roguelike/sais/sais-1.6.3~git.recipe [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+BUILD()
+{
+	mkdir sais-build && 
+	cmake -DCMAKE_INSTALL_PREFIX=./sais-build ../sais-$srcGitRev
```
Comment (Begasus) [inline games-roguelike/sais/sais-1.6.3~git.recipe:66]: Can't you use -DCMAKE_INSTALL_PREFIX="$appsDir/SaiS" (or simular)?

Comment (win8linux) [reply games-roguelike/sais/sais-1.6.3~git.recipe]: Yes, I've tested that now and it does work. That original one was based on the source's docs.

**Comment by extrowerk** [comment]
Builds fine, but the application have no icon yet.

**Comment by win8linux** [comment]
There is an icon for Palm OS and Windows Mobile ports which could be used as a basis for an icon:
image
However, I do not know how to make that the app icon nor figure out how to translate that into one compliant with the Haiku icon set. If the recipe is merged without an icon, then I'd make an issue about the missing icon soon afterwards.

**Pull request merged**